plotter/plot_datavalidation.py

```diff
@@ -148,20 +148,44 @@ def get_shape_sum(regions, process):
                 uncFile.ZG_PDF_met,
                 ]
             uncFile_photon = TFile(os.path.join(DIR,'../makeWorkspace/sys/photon_id_unc.root'))
+            uncFile_photon_scale = TFile(os.path.join(DIR,'../makeWorkspace/sys/photon_scale_unc.root'))
             if 'monojet' in category:
                 if year == 2017:
-                    uncertainties.append(uncFile_photon.monojet_2017_photon_id_extrap_up)
-                    uncertainties.append(uncFile_photon.monojet_2017_photon_id_up)
+                    h = uncFile_photon.monojet_2017_photon_id_extrap_up
+                    h.Scale(2)
+                    uncertainties.append(h)
+                    h = uncFile_photon.monojet_2017_photon_id_up
+                    h.Scale(2)
+                    uncertainties.append(h)
                 elif year == 2018:
-                    uncertainties.append(uncFile_photon.monojet_2018_photon_id_extrap_up)
-                    uncertainties.append(uncFile_photon.monojet_2018_photon_id_up)
+                    h = uncFile_photon.monojet_2018_photon_id_extrap_up
+                    h.Scale(2)
+                    uncertainties.append(h)
+                    h = uncFile_photon.monojet_2018_photon_id_up
+                    h.Scale(2)
+                    uncertainties.append(h)
+
+                h = uncFile_photon_scale.Get("photon_pt_scale_monojet_0.02_up")
+                h.Scale(0.5)
```

Comment:
I get a memory crash because of this line. Haven't found a fix yet, for now I'm using the 0.01 up histogram without any scaling